plugins/sentinelone/help.md

```diff
@@ -1715,327 +1656,448 @@ This trigger is used to get threats.
-|File Data|object|False|File data|
-|File Display Name|string|False|File display name|
-|File Extension Type|string|False|File extension type|
-|File is Dotnet|boolean|False|File is dotnet|
-|File is Executable|boolean|False|File is executable|
-|File is System|boolean|False|File is system|
-|File Malicious Content|boolean|False|File malicious content|
-|File Object ID|string|False|File object ID|
-|File Path|string|False|File path|
-|File SHA 256|string|False|File SHA 256|
-|File Verification Type|string|False|File verification type|
-|From Cloud|boolean|False|From cloud|
-|From Scan|boolean|False|From scan|
-|ID|string|False|ID|
-|In Quarantine|boolean|False|In quarantine|
-|Indicators|[]integer|False|Indicators|
-|Is Cert Valid|boolean|False|Is cert valid|
-|Is Interactive Session|boolean|False|Is interactive session|
-|Is Partial Story|boolean|False|Is partial story|
-|Malicious Group ID|string|False|Malicious group ID|
-|Malicious Process Arguments|string|False|Malicious process arguments|
-|Marked as Benign|boolean|False|Marked as Benign|
-|Mitigation Actions|[]string|False|Mitigation actions|
-|Mitigation Mode|string|False|Mitigation mode|
-|Mitigation Report|object|False|Mitigation report|
-|Mitigation Status|string|False|Mitigation status|
-|Publisher|string|False|Publisher|
-|Rank|integer|False|Rank|
-|Resolved|boolean|False|Resolved|
-|Site ID|string|False|Site ID|
-|Site Name|string|False|Site name|
-|Threat Agent Version|string|False|Threat agent version|
-|Threat Name|string|False|Threat name|
-|Updated At|string|False|Updated at|
-|Username|string|False|Username|
-|Whitening Options|[]string|False|Whitening options|
-
-#### pagination
-
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|Next Cursor|string|False|Next cursor|
-|Total Items|integer|False|Total items|
-
-#### quarantine_response
-
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|Data|object|False|Response data|
-|Errors|[]object|False|Errors|
 
 ## Troubleshooting
 
-To convert `threat` into an array use Type Converter Plugin
-For the Trigger settings, only set the Resolved field to False if solely resolved threats should be retrieved (i.e. setting to False will not include unresolved threats)
+* To convert `threat` into an array use Type Converter Plugin
+* For the Trigger settings, only set the Resolved field to False if solely resolved threats should be retrieved (i.e. setting to False will not include unresolved threats)
+* The Run Remote Script action may require starting a protected actions session to function properly. To do this, in the `code` input field, enter the passcode from a third-party app, such as Duo Mobile or Google Authenticator, set up in two-factor authentication. Entering the code is not required each time you run the action, because the session is valid for 30 minutes
 
 # Version History
 
+* 9.0.0 - Set `cloud_ready` to `true` in plugin spec | Rename URL input to Instance in connection | Code refactor
```

Comment:
Suggestion: Reword 'Set `cloud_ready` to `true` in plugin spec' -> Update plugin to allow cloud connections to be configured.